app/routes/api_route.py

```diff
@@ -0,0 +1,22 @@
+from flask import Blueprint, jsonify
+
+from app.services.user_service import UserService
+
+api_route = Blueprint("api", __name__)
+
+user_service = UserService()
```

Comment:
Potentially, there is scope to wrap this in a factory pattern and pass it through the `UserService` - but this works for now 👀